### ISSUE TITLE
Hyperlink colour with iOS Dark Mode

### ISSUE THREAD

**Issue by jbalint-de** [phedlund/CloudNews#3]
With active iOS Dark Mode links are coloured nearly the same as the background (dark blueish on black). A change to a way lighter colour may be necessary to optimize readability and accessibility.

**Comment by phedlund** [comment]
Fixed in version 3.11.0.
